modules/howtos/examples/user-manager.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+
+const couchbase = require("couchbase");
+
+async function go() {
+    const clusterAdm = new couchbase.Cluster("couchbase://localhost", {
```

Comment:
Probably best to use the new `couchbase.connect` method instead fo the deprecated `couchbase.Cluster` no?

So `await couchbase.connect("couchbase://localhost")`